frontend/website/src/Style.re

```diff
@@ -42,72 +42,125 @@ module Colors = {
 
 module Typeface = {
   open Css;
-  let weights = [
-    // The weights are intentionally shifted thinner one unit
-    (`thin, "Thin"),
-    (`extraLight, "Thin"),
-    (`light, "ExtraLight"),
-    (`normal, "Light"),
-    (`medium, "Regular"),
-    (`semiBold, "Medium"),
-    (`bold, "SemiBold"),
-    (`extraBold, "Bold"),
-  ];
-
-  // TODO: add format("woff") and unicode ranges
-  let () =
-    List.iter(
-      ((weight, name)) =>
-        ignore @@
-        fontFace(
-          ~fontFamily="IBM Plex Sans",
-          ~src=[
-            url("/static/font/IBMPlexSans-" ++ name ++ "-Latin1.woff2"),
-            url("/static/font/IBMPlexSans-" ++ name ++ "-Latin1.woff"),
-          ],
-          ~fontStyle=`normal,
-          ~fontWeight=weight,
-          (),
-        ),
-      weights,
-    );
-
-  let _ =
-    fontFace(
-      ~fontFamily="IBM Plex Mono",
-      ~src=[
-        url("/static/font/IBMPlexMono-Medium-Latin1.woff2"),
-        url("/static/font/IBMPlexMono-Medium-Latin1.woff"),
-      ],
-      ~fontStyle=`normal,
-      ~fontWeight=`semiBold,
-      (),
-    );
-  let _ =
-    fontFace(
-      ~fontFamily="IBM Plex Mono",
-      ~src=[
-        url("/static/font/IBMPlexMono-SemiBold-Latin1.woff2"),
-        url("/static/font/IBMPlexMono-SemiBold-Latin1.woff"),
-      ],
-      ~fontStyle=`normal,
-      ~fontWeight=`bold,
-      (),
-    );
-
-  let ibmplexserif =
-    fontFamily(
-      fontFace(
-        ~fontFamily="IBM Plex Serif",
-        ~src=[
-          url("/static/font/IBMPlexSerif-Medium-Latin1.woff2"),
-          url("/static/font/IBMPlexSerif-Medium-Latin1.woff"),
+  // To prevent "flash of unstyled text" on some browsers (firefox), we need
+  // to do insane things to mitigate it. Even though the CSS working group
+  // created `font-display: block` for this purpose, Firefox chooses to not
+  // follow the standard "wait for 3seconds before showing fallback fonts."
```

Comment:
According to the code it does: https://dxr.mozilla.org/mozilla-central/source/layout/style/nsFontFaceLoader.cpp#82

But in my firefox I definitely don't see _any_ block period whatsoever.

Reply:
Exactly

Reply:
Maybe it's fixed on master